app/src/main/java/com/android/unio/model/firestore/FirestoreReferenceList.kt

```diff
@@ -31,7 +31,7 @@ import kotlinx.coroutines.flow.StateFlow
  * @property collectionPath The path to the Firestore collection.
  * @property hydrate A function that converts a [DocumentSnapshot] to a [T].
  */
-class FirestoreReferenceList<T>(
+class FirestoreReferenceList<T : UniquelyIdentifiable>(
```

Comment:
Interesting addition, could you please explain to me what value this brings?

Reply:
This enables the `FirestoreReferenceList` to access the `uid` field of elements in its `list` property. 
Thus, it makes it possible to do for example the check I added to `.requestAll` that avoids excessive requests.